### PR TITLE
css-canvas-medical | Rocky Truong

### PR DESCRIPTION
**Pull request by rocky-truong**
image

### CODE REVIEW

**Comment by rocky-truong** [comment]
image

**Comment by rocky-truong** [comment]
image

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  width: 94%;
+  position: absolute;
+  opacity: 0.85;
+  height: 470px;
+}
+
+.hero-row {
+  display: flex;
+  position: relative;
+  margin-bottom: 530px;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:173]: Woah, that's a big one!! Why is this necessary? We may be able to solve this a different way

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+.hero-head {
+  position: absolute;
+  color: white;
+  top: 30px;
+  left: 120px;
+  width: 450px;
+  font-size: 36px;
+  font-weight: lighter;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:206]: Some big movements here, too. You might be able to reduce the overall changes by putting these elements in their own container, and just absolute positioning that one container

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this, dude!! A couple notes for you:

**Comment by rocky-truong** [comment]
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**